### ISSUE TITLE
In production, socket-io.client is missing

### ISSUE THREAD

**Issue by Stupidism** [michaelkrone/generator-material-app#70]
because server only serve this in development.

```
  var socket = socketio(server, {
    serveClient: (config.env !== 'production'),
    path: '/socket.io-client'
  });
```

I want to change this serveClient flag to always true. Is there a better way to fix this?

**Comment by Stupidism** [comment]
Seems to be fixed already in #62

**Comment by michaelkrone** [comment]
Yep, this is fixed with #62
